### DISCUSSION TITLE
Keyboard Shortcut Limitations

### DISCUSSION THREAD

**Discussion by Ichisich** [RamonUnch/AltDrag#375] [General]
It seems any keyboard combination which slightly resembles a native Windows shortcut gets disabled.

For example:
Win+Arrow
Win+(Alt/Ctrl/Shift)+Arrow 
Win+Home
...

I see the utility in that, but there should be an .ini entry to disable this behavior on account of the user's own risk.

**Comment by RamonUnch** [comment]
Unfortunately there is no way to overwrite an already installed global keyboard shortcut.
The only way would be to re-implement completely keyboard shortcut.
I am using the `RegisterHotKey()` native API.

[Clavier+](https://github.com/guilryder/clavier-plus) does that, also AHK does that.

**Reply by Ichisich** [reply]
I see.
So AltSnap automatically adapts to the used Windows version.
Gives the program some future-proofness.
